cmd/gen/makefile/flag.go

```diff
@@ -16,21 +16,29 @@ const (
 )
 
 type flag struct {
-	Flavour  string
-	Language string
+	Flavours gen.FlavourSlice
+	Language gen.Language
```

Comment:
Using custom values simplified the code inside runners nicely. Validation is now straightforward. But the important bit is that `Flavour  string ` is replaced with `Flavours gen.FlavourSlice`.